tracer/src/Datadog.Trace.OpenTracing/OpenTracingTracer.cs

```diff
@@ -42,7 +42,7 @@ public OpenTracingTracer(IDatadogTracer datadogTracer, global::OpenTracing.IScop
 
         public OpenTracingSpan ActiveSpan => (OpenTracingSpan)ScopeManager.Active?.Span;
 
-        ISpan ITracer.ActiveSpan => ScopeManager.Active?.Span;
+        global::OpenTracing.ISpan ITracer.ActiveSpan => ScopeManager.Active?.Span;
```

Comment:
Interesting that OpenTracing's API is `ActiveSpan` but ours it `ActiveScope`, since our API originally mirrored theirs (`Datadog.Trace.Tracer` implemented `OpenTracing.ITracer`).

It's not part of the official spec, so I guess it well under
>Optional API Elements
Some languages also provide utilities to pass an active Span and/or SpanContext around a single process.
(last section in https://opentracing.io/specification/)

Reply:
Sorry, just "thinking out loud"